examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -95,7 +101,7 @@ Where
 This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console.
 
 
-3. After the cluster is created successfully, perform the [post-install actions](../general/ami/post_installation.md) to configure the head node and compute nodes. 
+3. After the cluster is created successfully, [post-install actions](../general/ami/post_installation.md) now takes place automatically via `CustomActions` indicated in `launch.yaml` to configure the head node and compute nodes. 
 
```

Comment:
Add a section here to show customer they can check cluster status using describe-cluster command

pcluster describe-cluster -r us-west-2  -n My-ParallelCluster-Trn1

Reply:
Updated.

Reply:
updated in commit https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/14/commits/c882148c92ffa1fc67e69a8dd567b8c379a5a491